MMM-NINA.css

```diff
@@ -75,3 +75,38 @@
   font-size: 1.2em;
   font-weight: bold;
 }
+
+.nina-theme-top-floating.nina-wrapper {
+  position: absolute;
+  left: 50%;
+  transform: translate(-50%, 0);
+  z-index: 10;
+  opacity: 0.7;
+}
+
+.nina-theme-top-floating .nina-alert {
```

Comment:
wouldn't it make sense to use an or operator at the existing theme to avoid redundancy, as both themes are tight coupled?

Reply:
Sure. If this is possible. I'm not super fluent in CSS. I'll look into that.